### PR TITLE
Update rule_ids type in /v1/reports

### PR DESCRIPTION
**Pull request by MahanRahmati**

### CODE REVIEW

File: content/en/methods/reports.md [hunk 1]
```diff
@@ -57,7 +57,7 @@ category
 : String. Specify if the report is due to `spam`, `legal` (illegal content), `violation` of enumerated instance rules, or some `other` reason. Defaults to `other`. Will be set to `violation` if `rule_ids[]` is provided (regardless of any category value you provide).
 
 rule_ids[]
-: Array of Integer. For `violation` category reports, specify the ID of the exact rules broken. Rules and their IDs are available via [GET /api/v1/instance/rules]({{< relref "methods/instance#rules" >}}) and [GET /api/v1/instance]({{< relref "methods/instance#get" >}}).
+: Array of String. For `violation` category reports, specify the ID of the exact rules broken. Rules and their IDs are available via [GET /api/v1/instance/rules]({{< relref "methods/instance#rules" >}}) and [GET /api/v1/instance]({{< relref "methods/instance#get" >}}).
```
Comment (ThisIsMissEm) [inline content/en/methods/reports.md:60]: Might be worth referencing the ID docs? https://docs.joinmastodon.org/api/guidelines/#id

**Review by ThisIsMissEm — APPROVED** [review]
LGTM, one minor suggestion

**Review by ClearlyClaire — APPROVED** [review]

File: content/en/methods/reports.md (hunk 1 above)
Comment (MahanRahmati) [reply content/en/methods/reports.md]: Done!

**Review by ThisIsMissEm — APPROVED** [review]
LGTM

**Review by ClearlyClaire — APPROVED** [review]

**Pull request merged**